src/js/components/search/visualizations/rank/chart/ChartGroup.jsx

```diff
@@ -42,14 +44,26 @@ export default class ChartGroup extends React.Component {
         }
     }
 
+    processLink(label) {
+        let title = label;
+
+        if (this.props.linkID !== '') {
+            title = (<a href={`${this.props.urlRoot}${this.props.linkID}`}>
```

Comment:
Per https://developer.mozilla.org/en-US/docs/Web/SVG/Element/text, it appears `<a>` is a valid nested object of `<text>`